lib/crowbar/client/command/upgrade/repocheck.rb

```diff
@@ -49,6 +49,12 @@ def execute
                   err "No repochecks"
                 else
                   say formatter.result
+                  if provide_format == :table
+                    say "Please make sure that there are no missing repositories" \
+                      " before executing the next step."
```

Comment:
I'm not sure now, but I think that if there were missing repositories, step would fail, right? In such case, we do not need to tell user this, because he wouldn't be able to start new step anyway.